code/datums/components/chasm.dm

```diff
@@ -133,11 +134,48 @@
 			qdel(S.mmi)
 
 		falling_atoms -= AM
-		qdel(AM)
-		if(AM && !QDELETED(AM))	//It's indestructible
+		if (isliving(AM)) //living mobs get added to the contents of the chasm
 			var/atom/parent = src.parent
-			parent.visible_message(span_boldwarning("[parent] spits out [AM]!"))
+			AM.forceMove(parent)
 			AM.alpha = oldalpha
 			AM.color = oldcolor
 			AM.transform = oldtransform
-			AM.throw_at(get_edge_target_turf(parent,pick(GLOB.alldirs)),rand(1, 10),rand(1, 10))
+			var/mob/living/L = AM
+			L.Paralyze(100)
+			L.adjustBruteLoss(400)
```

Comment:
Bones should break on a fall like this, adjustBruteLoss does not roll for wounds

Reply:
give them a compond fracture on both legs